index.html

```diff
@@ -48,14 +48,14 @@ <h2>Haiku</h2>
                  <span style="color:black;background-color:#86BBD8;">
                 <br>On this road<br>
 
-                    On this road</br>
+                    On this road<br>
 
                         where nobody else travels
 
                     <br>autumn nightfall<br>
                     </span>
-                    <span style="background-color:#86BBD8;opacity: 0.5;">
-                by <b>Basho</b>
+                    <span class="byline">
+                by <b>Basho</b></span>
```

Comment:
you were supposed to use a class here and the rely on the stylesheet to apply the style